xc/domain.ml

```diff
@@ -1066,9 +1066,18 @@ let restore_vgpu (task: Xenops_task.t) ~xc ~xs domid fd vgpu vcpus =
 
 type suspend_flag = Live | Debug
 
-let write_libxc_record (task: Xenops_task.t) ~xc ~xs ~hvm xenguest_path domid uuid fd flags progress_callback qemu_domid do_suspend_callback =
+let write_libxc_record (task: Xenops_task.t) ~xc ~xs ~hvm xenguest_path domid uuid fd vgpu_fd flags progress_callback qemu_domid do_suspend_callback =
```

Comment:
I'm slightly uncomfortable with having a string of nine unlabelled parameters to be supplied in the right order, especially with two consecutive params having the same type (`fd` and `vgpu_fd`).